var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -78,10 +81,13 @@ def install(self, spec, prefix):
             ])
 
         # BLAS arguments require path to libraries
-        # FIXME : (blas / lapack always provide libblas and liblapack as aliases)
-        make_args.extend([
-            'BLAS=-lblas',
-            'LAPACK=-llapack'
-        ])
+        # FIXME: (blas/lapack always provide libblas and liblapack as aliases)
+        if '@4.5.1' in spec:
```

Comment:
the make_args.extend was in the original suite-sparse package for version 4.5.1, so I'm not sure if we want to change that and risk breaking previous users. I'm also not confident that the suite-sparse build system will obey the to_link_flags. The 4.5.3 version does not require these flags (I think they fixed their makefile).